pages/process.js

```diff
@@ -0,0 +1,159 @@
+import Link from "next/link";
+
```

Comment:
Please combine the info on this page into the FAQ.

Reply:
I don't see the need for this to be a whole separate file/page, it just adds complexity.

Reply:
but the FAQ page shouldn't have it because it is an FAQ page.

Reply:
It's too much information for one page

Reply:
if you have a better way of going about this let me know

Reply:
I don't think we need another page, please combine / reduce the info so it can be added to FAQ.

Reply:
Done